README.md

```diff
@@ -27,6 +27,9 @@ noInstall does not run 'npm install', only 'npm run build'. If node modules are
 This flag is set in the following manner: -D [flag]=[value]
 The script logic only looks for the two option flags to be set, the value does not matter.
 
+To build zlux-core with multiple pull request edit the Jenkinsfile to point to the pull request you want to build with. There should be a map call zluxParameters with keys and values, just
```

Comment:
you need to elaborate and put this toward the top so people dont miss it. such as

## Building a zlux-core release
This repo contains the jenkinsfile required for building zlux-core.pax used in zowe release pipelines.
By default, this builds each zlux repo from the staging branch.
If you are making changes to multiple zlux repos, you must edit the jenkinsfile (explain how) and submit a PR here in order to generate a build that includes all PRs you want.